apps/mosaico/src/Mosaico/Header/Menu.purs

```diff
@@ -265,10 +271,27 @@ render { props: { visible } } = DOM.div
         renderSection :: Section -> JSX
         renderSection { modifier, subsections, title } = DOM.div
           { className: unwords [ sectionClass, sectionClass <> modifier ]
-          , children: [ DOM.div { className: sectionTitleClass
+          , children: [ DOM.div
+                          { className: sectionHeaderClass
+                          , children:
+                              DOM.div
+                                { className: sectionTitleClass
                                 , children: [ DOM.text title ]
-                                }
-                      ] <> (renderSubsection <$> subsections)
+                                } `cons` 
+                                  if length subsections > 0 then
```

Comment:
We could use `guard` from `Data.Monoid` here:

```purescript
guard (not $ null subsections)
  DOM.div { className: sectionExpanderClass }
```
This is a matter of personal preference of course, but we could also ditch the `snoc` and construct the array like:
```purescript
-- Instead of
...
, children:
    DOM.div
      { className: sectionTitleClass
      , children: [ DOM.text title ]
      } `cons`
      guard (length subsections > 0)
        [ DOM.div { className: sectionExpanderClass  } ]

-- why not
...
, children
   [ DOM.div
      { className: sectionTitleClass
      , children: [ DOM.text title ]
      }
    , guard (length subsections > 0)
        DOM.div { className: sectionExpanderClass  }
    ]
```

Empty `JSX's` does not affect the DOM, meaning `[ DOM.text "yolo", mempty ]` would render only `[ DOM.text "yolo" ]`.